routes/dependencies/remove/dependency.js

```diff
@@ -4,7 +4,9 @@ const ansiCommandStream = require('../../../src/ansiCommandStream');
 module.exports = async (req, res) => {
   const { params: { dependency } } = req;
   try {
-    const { stream, kill } = ansiCommandStream({ args: ['rm', dependency] });
+    const { stream, kill } = ansiCommandStream({
+      args: ['remove', dependency],
```

Comment:
this should be `rm` so it works for yarn and npm (or we can have a mapping at ansiCommandStream)

Reply:
I tried running that with "rm" and it doesn't work

Reply:
If i remember, "yarn rm" is not a correct yarn command. Correct me if i'm wrong

Reply:
Okay, let’s do that on the server 👌